src/node/node_to_node.h

```diff
@@ -209,9 +209,16 @@ namespace ccf
       size_t size) override
     {
       auto n2n_channel = channels->get(to);
-      // Sending after a channel has been destroyed is a bug.
-      assert(n2n_channel);
-      return n2n_channel->send(type, {data, size});
+      if (!n2n_channel)
+      {
+        LOG_DEBUG_FMT(
```

Comment:
Curious: why has this changed in this PR?

Reply:
That's because BFT assumes that it can reach all the nodes at all times and this was the quickest way to make it work. I'll take another look at this after the ongoing changes and if possible I'd like to make it a failure again.

Reply:
In what circumstances are we attempting to send to an unknown node?